### ISSUE TITLE
range of values for image?

### ISSUE THREAD

**Issue by unhandyandy** [anishathalye/neural-style#84]
In stylize the default initialization of the image is by
`
initial = tf.random_normal(shape) * 0.256`

and

`        image = tf.Variable(initial)
`

I saved this initial image and had a look, but it doesn't seem to contain any randomness.  I tried 

`initial = tf.random_normal(shape) * 0.256*4e2`

and that seems to produce visible randomness.  What is the proper range of values for the elements in image?  Are they in [-1,1] or [-127,128], or something else?

**Comment by anishathalye** [comment]
Uhh I don't really remember. I think the image is something like [-127,127]. The code does some clipping and stuff.

Also, some of this stuff was changed in commits like ab0ad031e8ce1be0be1a1e6b6b97d0eb9ad93753 (which are pull requests, and it's been a while, so I don't really remember the merged code). Seems like there's some weird stuff going on, e.g. `noise = ...` (in stylize.py) is an unused variable.

**Comment by unhandyandy** [comment]
I also notice that the default ADAM learning rate seems high at 10; I'm new to this but most sources seem to recommend a value of around 0.01.  I wonder if adjusting this and the randomness in the initialization could speed up the training?

**Comment by anishathalye** [comment]
The choice of learning rate depends on the neural net structure / problem domain - there's no such thing as "the recommended value for learning rate" that's universally applicable to all nets (if there were, we wouldn't have a tunable learning rate).

Back when I wrote this, I did some experiments with learning rate and found that my choice works reasonably well with the input data I tried.

If you want to experiment with some different learning rates and with different types of initialization, that would be cool. If you discover something that works better, you could send a pull request.

**Comment by unhandyandy** [comment]
I meant that other implementations of neural style used much lower rates.  I'll play around with it.

**Comment by anishathalye** [comment]
Ahh, okay.

Sounds good! If you find something, let me know :)

**Comment by unhandyandy** [comment]
Not finding any silver bullets.
